### ISSUE TITLE
Numerals between 11-19

### ISSUE THREAD

**Issue by shirawigi** [IAHLT/UD_Hebrew#51]
Hi @amir-zeldes,

In [issue 12](https://github.com/IAHLT/UD_Hebrew/issues/12) a question about numbers between 11-19 was raised as to how we should annotate them when they appear as Hebrew word rather than as a Arabic figure, in both a NUM-cardinal and in an ADJ-ordinal position. 
The answer there was to assign deprel between the two tokens composing the whole number, as follows:
הכנסת השלוש עשרה
compound(shlosh, esre)
Each token is an ADJ in this case, and I assume that for the cardinal case it would have the same deprel, with UPOS = NUM.

The reasoning for this was that Shlosh-esre presents a construct structure. 
I think it is deceiving, and that it is not a real construct structure, because if we look at the male form, we have האפיפיור השלושה-עשר, and not האפיפיור השלושת-עשר as we would expect if it were a real construct structure.

Now, I consider shlosh esre (and the rest of the numbers) to be similar to a proper noun, as it is a name of a number. And just like other proper nouns, I expect the deprel between them to represent a flat structure rather than a hierarchical one (suggested by the compound deprel).
Therefore, I suggest that the deprel between the two tokens would be ‘flat’.
This, I believe, represents better the relation between the two ways of writing numbers, in words or figures, as 13 is a single token, and a flat deprel between shlosh and esre makes it clear that the two tokens are considered a single unit in the speaker’s mind.

What do you think?
Thanks!

**Comment by amir-zeldes** [comment]
I think the `compound` suggestion might go back to English, and not a construct state; in UD English, complex numbers spelled out, like "four thousand", are annotated as compound, which is a carryover from how they were analyzed in Stanford Dependencies (before UD).

I'm not hugely opposed to using `flat` here, but using `compound` is not terrible either from the perspective that שלוש is a normal number, and the form "asar" or "esre" is a kind of affixal form that is not normally used by itself, so you could say it's a modifier added in order to create some kind of compound number. Incidentally I think we did the same in Coptic (compound), but probably again mainly inspired by English and the idea that this is in some way a morphologically complex, but single 'word'.

If you want to go with flat, maybe it's worth taking an inventory of complex number expressions and thinking which ones should be flat - you can take a look at the "torture example" @nschneid came up with in section 8 of this paper: https://arxiv.org/pdf/2108.12928.pdf

**Comment by shirawigi** [comment]
Hi @amir-zeldes,

I read the torture example... torturing indeed :)
I guess I agree with the compositional analysis suggested there. Thanks for the reference.
But now, I'm afraid we would have to change the way we treated "7 Million..." I think that until now we annotated "7 מיליון" and "4 אלפים" with nummod rather than compound, as suggested in the paper.

ex. for [7M](https://grew-match.iahlt.org/?corpus=htb-new&custom=61e7128192904)
ex. for [10K](https://grew-match.iahlt.org/?corpus=htb-new&custom=61e713aaba563)

Would you say we should change that?

Thanks a lot,
Shira

**Comment by amir-zeldes** [comment]
No, actually I think it's OK for Hebrew as nummod - the point in English is that plural inflection is missing (not "four hundreds people") so it behaves like nominal compounding in that respect. But ארבעת אלפים is actually regular, so I think you did it right.
